composer.lock

```diff
@@ -227,17 +227,17 @@
         },
         {
             "name": "cpsit/project-builder-shared-ddev",
-            "version": "1.0.1",
+            "version": "1.2.1",
```

Comment:
A lot of dependencies have been updated. Can you please reset `composer.lock` and run `composer update phpstan/phpstan` only? We can update all other dependencies in a separate patch.

Reply:
That should not have happened. I do apologise sincerely.